io.openems.edge.controller.byd.alarm/bnd.bnd

```diff
@@ -0,0 +1,27 @@
+Bundle-Name: OpenEMS Edge Controller BydAlarm
```

Comment:
Make sure to align this file with the other bnd.bnd files; e.g. "buildpath" and "testpath" need to be different.